src/Underworld.ts

```diff
@@ -2778,10 +2778,6 @@ ${CardUI.cardListToImages(player.stats.longestSpell)}
                 }
                 return '';
               }
-              if (stat == 'attackRange') {
-                // Display default attackRange as "100"
-                return Math.floor(100 * value / config.PLAYER_BASE_ATTACK_RANGE);
-              }
```

Comment:
Why was this removed?

Reply:
I think this makes sense in the context of the new stat values, essentially it's a lot easier to go over 200 displayed cast range, and at that point having the display divided by 2 may just be more confusing / more calculation. You don't actually know what your cast range is relative to stamina and enemy attack ranges because the displayed value is arbitrarily divided by 2.

I did see the feedback on discord from a while back saying that "240 cast range doesn't mean anything to me", but they were asking for range in terms of "tiles", and the current 1/2 display doesn't really solve that feedback, it just makes the number smaller. At least now it would have some meaning/reference in the game world

Reply:
Works for me!